p3dopenvr/p3dopenvr.py

```diff
@@ -22,8 +22,9 @@ def exitfunc():
     openvr.shutdown()
 
 class P3DOpenVR():
-    def __init__(self):
+    def __init__(self,verbose):
```

Comment:
Could you add a space after the comma and set the default value for verbose to True